fontforge/featurefile.c

```diff
@@ -509,9 +509,6 @@ return( NULL );
 		/* !!! Bug. Lose device tables here */
 		if ( isv )
 		    space->u.pair.vr[0].v_adv_off = kp->off;
```

Comment:
There's probably an `else` missing here, and if not the indentation should be adjusted.

Reply:
Right, deleted one line too many.